### DISCUSSION TITLE
3D SMOKE

### DISCUSSION THREAD

**Discussion by WYC68** [firemodels/fds#12688] [Q&A]
您好 我是個新手
我嘗試建立一個空間要模擬煙霧 但我的表單內沒有3D SMOKE 選項
![S__145686576](https://github.com/firemodels/fds/assets/165125646/0a149cf6-d337-4c5a-9984-2aa3ecf7850b)
請問是哪個環節有問題呢

**Comment by rmcdermo** [comment]
Welcome to the forum.  But please use English, as most of us do not know Chinese.  Please do the translation on your end.

From Google Translate:
"Hello I am a newbie I'm trying to create a space to simulate smoke but there is no 3D SMOKE option in my form"

My guess is that you have not entered a soot yield.  Please read the FDS user guide and search on SOOT_YIELD.

**Reply by WYC68** [reply]
Thank you!!!
I have remodeled, but the same problem occurs. I am not sure if there is something wrong with my parameters. Can you please help me take a look?

&HEAD CHID='Example0325', TITLE='Example6.8.0'/
&MESH XB=0,35,0,11.4,0,42.75
                 IJK=360,120,432/

&VENT MB='XMAX',SURF_ID='OPEN'/
&VENT MB='YMAX',SURF_ID='OPEN'/
&VENT MB='ZMAX',SURF_ID='OPEN'/

&TIME T_END=1200/
&DUMP NFRAMES=800,DT_HRR=10,DT_DEVC=20/

&MISC TMPA=30/ 

&SURF ID          = 'WALL'
        COLOR     = 'GRAY'
MATL_ID   = 'CONCRETE'
        THICKNESS = 0.1                                                                                                                                                                                                                    
        DEFAULT = .TRUE./    

&MATL ID            = 'CONCRETE '
      FYI           = 'Quintiere, Fire Behavior'
      SPECIFIC_HEAT = 0.88                            
      DENSITY       = 2100.                
      CONDUCTIVITY  = 1.0 / 
         
&OBST XB=32.0,33.0,2.0,3.0,3.75,3.8, SURF_ID='Fire'/
&SURF ID='Fire', HRRPUA=1000, TAU_Q=-100,  COLOR='RED'/


&REAC FUEL='PROPANE'
               SOOT_YIELD=0.05/  



&VENT XB=28,33,1.6,2.8,6.65,6.65,SURF_ID=VENT1/
&SURF ID='VENT1',VEL=2,TAU_Q=-60/


===Wall===
=1F=
&OBST XB=1.0,30.0,1.0,1.2,0,3.75/1
&OBST XB=1.0,1.1,5.5,5.9,0,3.75/2
&OBST XB=1.0,30.0,10.2,10.4,0,3.75/3
&OBST XB=29.8,30.0,1.2,4.2,0,3.75/4
&OBST XB=10.9,30.0,4.2,4.4,0,3.75/5
&OBST XB=23.8,24.0,1.2,4.2,0,3.75/6
&OBST XB=22.0,22.2,1.2,4.2,0,3.75/7
&OBST XB=22.2,23.8,1.7,1.9,0,3.75/8
&OBST XB=22.2,23.8,3.5,3.7,0,3.75/9
&OBST XB=19.7,19.9,1.2,4.2,0,3.75/10
&OBST XB=16.8,17.0,1.2,4.2,0,3.75/11
&OBST XB=14.0,14.2,1.2,4.2,0,3.75/12
&OBST XB=10.9,11.1,4.4,6.3,0,3.75/13
&OBST XB=2.5,11.1,6.3,6.5,0,3.75/14
&OBST XB=2.5,2.7,1.2,6.3,0,3.75/15
&OBST XB=9.1,9.3,7.9,10.2,0,3.75/16
&OBST XB=9.1,12.3,7.7,7.9,0,3.75/17
&OBST XB=12.3,12.5,6.1,10.2,0,3.75/18
&OBST XB=12.3,18.7,5.9,6.1,0,3.75/19
&OBST XB=15.4,15.6,6.1,10.2,0,3.75/20
&OBST XB=18.5,18.7,6.1,10.2,0,3.75/21
&OBST XB=22.0,22.2,5.9,10.2,0,3.75/22
&OBST XB=22.2,24.5,5.9,6.1,0,3.75/23
&OBST XB=24.5,24.7,5.9,10.2,0,3.75/24
&OBST XB=33.5,34.0,1.0,1.5,0,3.75/25
&OBST XB=33.5,34.0,9.9,10.4,0,3.75/26

=2F=
&OBST XB=9.0,34.0,1.0,1.2,3.75,6.9/1
&OBST XB=10.2,10.4,1.2,10.2,3.75,6.9/2
&OBST XB=9.0,34.0,10.2,10.4,3.75,6.9/3
&OBST XB=33.8,34.0,1.2,10.2,3.75,6.9/4
&OBST XB=31.3,33.8,5.6,5.8,3.75,6.9/5
&OBST XB=31.1,31.3,1.2,10.2,3.75,6.9/6
&OBST XB=16.6,31.1,4.2,4.4,3.75,6.9/7
&OBST XB=29.4,29.6,1.2,4.2,3.75,6.9/8
&OBST XB=23.8,24.0,1.2,4.2,3.75,6.9/9
&OBST XB=22.2,23.8,1.7,1.9,3.75,6.9/10
&OBST XB=22.2,23.8,3.5,3.7,3.75,6.9/11
&OBST XB=22.0,22.2,1.2,4.2,3.75,6.9/12
&OBST XB=19.7,19.9,1.2,4.2,3.75,6.9/13
&OBST XB=16.4,16.6,1.2,7.1,3.75,6.9/14
&OBST XB=10.4,16.4,5.9,6.1,3.75,6.9/15
&OBST XB=15.4,15.6,7.1,10.2,3.75,6.9/16
&OBST XB=15.6,22.0,7.1,7.3,3.75,6.9/17
&OBST XB=18.5,18.7,7.3,10.2,3.75,6.9/18
&OBST XB=22.0,22.2,7.1,10.2,3.75,6.9/19
&OBST XB=9.0,9.2,1.2,10.2,3.75,4.85/20
&OBST XB=1.0,34.0,1.0,10.4,3.55,3.75/1F天花、2F地板

=3F=
&OBST XB=9.0,34.0,1.0,1.2,6.9,10.05/1
&OBST XB=10.2,10.4,1.2,10.2,6.9,10.05/2
&OBST XB=9.0,34.0,10.2,10.4,6.9,10.05/3
&OBST XB=33.8,34.0,1.2,10.2,6.9,10.05/4
&OBST XB=17.2,33.8,4.2,4.4,6.9,10.05/5
&OBST XB=29.4,29.6,1.2,4.2,6.9,10.05/6
&OBST XB=23.8,24.0,1.2,4.2,6.9,10.05/7
&OBST XB=22.0,22.2,1.2,4.2,6.9,10.05/8
&OBST XB=22.2,23.8,1.7,1.9,6.9,10.05/9
&OBST XB=22.2,23.8,3.5,3.7,6.9,10.05/10
&OBST XB=19.7,19.9,1.2,4.2,6.9,10.05/11
&OBST XB=17.0,17.2,1.2,7.2,6.9,10.05/12
&OBST XB=10.4,17.0,5.3,5.5,6.9,10.05/13
&OBST XB=17.2,18.0,7.2,7.4,9.05,10.05/14
&OBST XB=18.0,18.2,7.2,10.2,6.9,10.05/15
&OBST XB=18.2,23.0,7.2,7.4,6.9,10.05/16
&OBST XB=23.0,23.2,7.2,10.2,6.9,10.05/17
&OBST XB=26.2,26.4,7.2,10.2,6.9,10.05/18
&OBST XB=26.4,30.5,7.2,7.4,6.9,10.05/19
&OBST XB=29.4,29.6,7.4,10.2,6.9,10.05/20
&OBST XB=30.5,30.7,4.4,7.4,6.9,10.05/21
&OBST XB=9.0,9.2,1.2,10.2,6.9,8.0/22
&OBST XB=9.0,34.0,1.0,10.4,6.7,6.9/2F天花、3F地板

===Opening===
=1F=
&HOLE XB=29.7,30.1,2.6,4.2,0,2.5/1
&HOLE XB=23.7,24.1,2.2,3.2,0,2.5/2
&HOLE XB=21.9,22.3,7.1,8.1,0,2.1/3
&HOLE XB=19.9,20.8,4.1,4.5,0,2.1/4
&HOLE XB=17.0,17.9,4.1,4.5,0,2.1/5
&HOLE XB=14.2,15.1,4.1,4.5,0,2.1/6
&HOLE XB=13.1,14.0,4.1,4.5,0,2.1/7
&HOLE XB=12.2,12.6,7.9,8.8,0,2.1/8
&HOLE XB=14.5,15.4,5.8,6.2,0,2.1/9
&HOLE XB=15.6,16.5,5.8,6.2,0,2.1/10

=2F=
&HOLE XB=33.7,34.1,1.6,2.8,4.75,5.95/1
&HOLE XB=33.7,34.1,3.0,4.2,4.75,5.95/2
&HOLE XB=33.7,34.1,4.4,5.6,4.75,5.95/3
&HOLE XB=33.7,34.1,5.8,7.0,4.75,5.95/4
&HOLE XB=33.7,34.1,7.2,8.4,4.75,5.95/5
&HOLE XB=33.7,34.1,8.6,9.8,4.75,5.95/6
&HOLE XB=31.0,31.4,4.4,5.6,3.75,5.85/7
&HOLE XB=31.0,31.4,5.8,7.0,3.75,5.85/8
&HOLE XB=29.6,30.6,4.3,4.5,3.75,5.85/9
&HOLE XB=24.0,25.2,4.3,4.5,3.75,6.25/10
&HOLE XB=23.7,24.1,2.2,3.2,3.75,6.25/11
&HOLE XB=19.9,20.8,4.3,4.5,3.75,5.85/12
&HOLE XB=18.7,19.7,4.3,4.5,3.75,5.85/13
&HOLE XB=16.3,16.7,4.9,5.9,3.75,5.85/14
&HOLE XB=10.1,10.5,2.8,4.8,3.75,6.25/15
&HOLE XB=10.1,10.5,6.6,8.6,3.75,6.25/16
&HOLE XB=16.3,16.7,6.1,7.1,3.75,5.85/17
&HOLE XB=17.5,18.5,7.1,7.4,3.75,5.85/18
&HOLE XB=18.7,19.7,7.0,7.4,3.75,5.85/19

=3F=
&HOLE XB=33.7,34.1,1.6,2.8,7.9,9.1/1
&HOLE XB=33.7,34.1,3.0,4.2,7.9,9.1/2
&HOLE XB=33.7,34.1,4.4,5.6,7.9,9.1/3
&HOLE XB=33.7,34.1,5.8,7.0,7.9,9.1/4
&HOLE XB=33.7,34.1,7.2,8.4,7.9,9.1/5
&HOLE XB=33.7,34.1,8.6,9.8,7.9,9.1/6
&HOLE XB=29.6,30.5,4.1,4.5,6.9,9.0/7
&HOLE XB=29.6,30.5,7.1,7.5,6.9,9.0/8
&HOLE XB=28.5,29.4,7.1,7.5,6.9,9.0/9
&HOLE XB=24.0,25.6,4.1,4.5,6.9,9.4/10
&HOLE XB=23.7,24.1,2.2,3.2,6.9,9.4/11
&HOLE XB=19.9,20.8,4.3,4.5,6.9,9.0/12
&HOLE XB=17.2,18.1,4.1,4.5,6.9,9.0/13
&HOLE XB=16.9,17.3,4.4,5.3,6.9,9.0/14
&HOLE XB=10.1,10.5,2.8,4.8,6.9,9.4/15
&HOLE XB=10.1,10.5,6.6,8.6,6.9,9.4/16
&HOLE XB=22.0,23.0,7.1,7.5,6.9,9.4/17

**Comment by mcgratta** [comment]
Your MESH has 18.6 million cells. Are you sure that the case ran? Did the job complete?

**Reply by WYC68** [reply]
Sorry, because I learned from watching online videos, and I don’t understand many things very well! So I can only keep trying to correct the mistakes, because I can’t make the model burn, so I would like to ask you which parameter I have a problem with, and my Is the concept wrong?

**Comment by drjfloyd** [comment]
Your grid is 0.09896 m in z.

&MESH XB=0,35,0,11.4,0,42.75
IJK=360,120,432/

here is your fire OBST

&OBST XB=32.0,33.0,2.0,3.0,3.75,3.8, SURF_ID='Fire'/

These z values sit at 37.9 and 38.4 grid cells. Both will snap to 38 grid cells.

Here is your floor OBST
&OBST XB=1.0,34.0,1.0,10.4,3.55,3.75/

These z value sit at 35.9 and 38.4 grid cells and will snap to 36 and 38 grid cells.

Note the floor (which is defined second) has the same upper grid cell boundary as the fire after snapping to the grid. Per the User's Guide:

7.2.4 Overlapping Obstructions

If the faces of two obstructions overlap each other, FDS will choose the surface properties of the obstruction
that is specified second in the input file.

**Reply by WYC68** [reply]
So do I need to reset the position of my fire source?
Will there be any problems with my intake and exhaust settings?

**Comment by drjfloyd** [comment]
Make sure you have read the FDS documentation. I also suggest you study the many example files provided to you when you install FDS. All of the examples in the User's Guide and the Verfication Guide are in the Examples folder where you installed FDS.

Also suggest if you are learning FDS to start with very simple problems - an input with just a fire, an input with just supply and exahust, etc. Leaping straight to a large complex input file means if you make multiple mistakes it can be hard to figure out on your own what the problem is. 

Your problem is the fire OBST and the floor OBST are occupying the same grid cell and per the User's Guide the second OBST (the floor) takes precendence.  Moving the fire is one option to fix this.

For your intake and exhaust: are they working as you expect? If not, then you have a problem with the inputs.
